packages/rich-text-editor/test/a11y.common.js

```diff
@@ -178,15 +178,15 @@ describe('accessibility', () => {
     it('should move focus to next element after esc followed by tab are pressed', async () => {
       const wrapper = fixtureSync(`<div>
         <vaadin-rich-text-editor></vaadin-rich-text-editor>
-        <button>button</button>
+        <input id="last-global-focusable" />
```

Comment:
I replaced `<button>` with `<input>` to fix a local failure in Webkit. For some reason, buttons aren't always tab-focusable in Webkit on Mac OS.